pom.xml

```diff
@@ -103,7 +103,7 @@
         <org.slf4j.version>1.7.32</org.slf4j.version>
         <org.apache.poi.version>5.1.0</org.apache.poi.version>
         <org.bouncycastle.version>1.69</org.bouncycastle.version>
-        <com.fasterxml.jackson.version>2.11.4</com.fasterxml.jackson.version>
+        <com.fasterxml.jackson.version>2.13.0</com.fasterxml.jackson.version>
```

Comment:
Does StoreLogData ITEST works properly with new jackson version? As far as I remember, there were some errors in logs

Reply:
It has been approved by me. Choice between not working test on Java 11 and critical requested improvement, I select the second.

Reply:
When I run tests on my machine, the only test that was failed, was itest for EPBDS-7947, I fixed error message for that.
If StoreLogData will fail, probably we will drop elastic search support. Please refer to @yurkom .

Reply:
ok, I don't know why tests are passed on my jdk-11, but on CI in github StoreLogData is failed.
I will add @ Ignore to that test.

Reply:
I'm talkina about the following error:
````
org.openl.rules.ruleservice.storelogdata.StoreLogDataException: Failed on ElasticSearch entity save operation.
	at org.openl.rules.ruleservice.storelogdata.elasticsearch.ElasticSearchStoreLogDataService.save(ElasticSearchStoreLogDataService.java:154) ~[org.openl.rules.ruleservice.ws.storelogdata.elasticsearch-5.26.0-SNAPSHOT.jar:5.26.0-SNAPSHOT]
	at org.openl.rules.ruleservice.storelogdata.AbstractStoreLogDataService.save(AbstractStoreLogDataService.java:19) ~[org.openl.rules.ruleservice.ws.storelogdata-5.26.0-SNAPSHOT.jar:?]
	at org.openl.rules.ruleservice.storelogdata.SimpleStoreLogDataManager.store(SimpleStoreLogDataManager.java:34) ~[webservice.jar:?]
	at org.openl.rules.ruleservice.kafka.publish.KafkaService.lambda$sendErrorToDlt$2(KafkaService.java:408) ~[webservice.jar:?]
	at org.apache.kafka.clients.producer.KafkaProducer$InterceptorCallback.onCompletion(KafkaProducer.java:1365) ~[kafka-clients-2.8.1.jar:?]
	at org.apache.kafka.clients.producer.internals.ProducerBatch.completeFutureAndFireCallbacks(ProducerBatch.java:228) ~[kafka-clients-2.8.1.jar:?]
	at org.apache.kafka.clients.producer.internals.ProducerBatch.done(ProducerBatch.java:197) ~[kafka-clients-2.8.1.jar:?]
	at org.apache.kafka.clients.producer.internals.Sender.completeBatch(Sender.java:684) ~[kafka-clients-2.8.1.jar:?]
	at org.apache.kafka.clients.producer.internals.Sender.completeBatch(Sender.java:655) ~[kafka-clients-2.8.1.jar:?]
	at org.apache.kafka.clients.producer.internals.Sender.lambda$null$1(Sender.java:574) ~[kafka-clients-2.8.1.jar:?]
	at java.util.ArrayList.forEach(ArrayList.java:1259) ~[?:1.8.0_312]
	at org.apache.kafka.clients.producer.internals.Sender.lambda$handleProduceResponse$2(Sender.java:561) ~[kafka-clients-2.8.1.jar:?]
	at java.lang.Iterable.forEach(Iterable.java:75) ~[?:1.8.0_312]
	at org.apache.kafka.clients.producer.internals.Sender.handleProduceResponse(Sender.java:561) ~[kafka-clients-2.8.1.jar:?]
	at org.apache.kafka.clients.producer.internals.Sender.lambda$sendProduceRequest$3(Sender.java:785) ~[kafka-clients-2.8.1.jar:?]
	at org.apache.kafka.clients.ClientResponse.onComplete(ClientResponse.java:109) [kafka-clients-2.8.1.jar:?]
	at org.apache.kafka.clients.NetworkClient.completeResponses(NetworkClient.java:584) [kafka-clients-2.8.1.jar:?]
	at org.apache.kafka.clients.NetworkClient.poll(NetworkClient.java:576) [kafka-clients-2.8.1.jar:?]
	at org.apache.kafka.clients.producer.internals.Sender.runOnce(Sender.java:327) [kafka-clients-2.8.1.jar:?]
	at org.apache.kafka.clients.producer.internals.Sender.run(Sender.java:242) [kafka-clients-2.8.1.jar:?]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
Caused by: org.springframework.data.elasticsearch.ElasticsearchException: failed to index the document [id: f3321fb0-567a-11ec-84c7-45b67eb249e2]
	at org.springframework.data.elasticsearch.core.ElasticsearchRestTemplate.prepareIndex(ElasticsearchRestTemplate.java:1463) ~[spring-data-elasticsearch-3.2.13.RELEASE.jar:3.2.13.RELEASE]
	at org.springframework.data.elasticsearch.core.ElasticsearchRestTemplate.index(ElasticsearchRestTemplate.java:701) ~[spring-data-elasticsearch-3.2.13.RELEASE.jar:3.2.13.RELEASE]
	at org.openl.rules.ruleservice.storelogdata.elasticsearch.ElasticSearchStoreLogDataService.save(ElasticSearchStoreLogDataService.java:151) ~[org.openl.rules.ruleservice.ws.storelogdata.elasticsearch-5.26.0-SNAPSHOT.jar:5.26.0-SNAPSHOT]
	... 20 more
Caused by: com.fasterxml.jackson.databind.exc.InvalidDefinitionException: Java 8 date/time type `java.time.ZonedDateTime` not supported by default: add Module "com.fasterxml.jackson.datatype:jackson-datatype-jsr310" to enable handling (through reference chain: org.openl.itest.elasticsearch.CustomElasticEntity2["incomingTime"])
	at com.fasterxml.jackson.databind.exc.InvalidDefinitionException.from(InvalidDefinitionException.java:77) ~[jackson-databind-2.13.0.jar:2.13.0]
	at com.fasterxml.jackson.databind.SerializerProvider.reportBadDefinition(SerializerProvider.java:1300) ~[jackson-databind-2.13.0.jar:2.13.0]
	at com.fasterxml.jackson.databind.ser.impl.UnsupportedTypeSerializer.serialize(UnsupportedTypeSerializer.java:35) ~[jackson-databind-2.13.0.jar:2.13.0]
	at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:728) ~[jackson-databind-2.13.0.jar:2.13.0]
	at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:774) ~[jackson-databind-2.13.0.jar:2.13.0]
	at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:178) ~[jackson-databind-2.13.0.jar:2.13.0]
	at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider._serialize(DefaultSerializerProvider.java:480) ~[jackson-databind-2.13.0.jar:2.13.0]
	at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider.serializeValue(DefaultSerializerProvider.java:319) ~[jackson-databind-2.13.0.jar:2.13.0]
	at com.fasterxml.jackson.databind.ObjectMapper._writeValueAndClose(ObjectMapper.java:4569) ~[jackson-databind-2.13.0.jar:2.13.0]
	at com.fasterxml.jackson.databind.ObjectMapper.writeValueAsString(ObjectMapper.java:3822) ~[jackson-databind-2.13.0.jar:2.13.0]
	at org.springframework.data.elasticsearch.core.DefaultEntityMapper.mapToString(DefaultEntityMapper.java:77) ~[spring-data-elasticsearch-3.2.13.RELEASE.jar:3.2.13.RELEASE]
	at org.springframework.data.elasticsearch.core.ElasticsearchRestTemplate.prepareIndex(ElasticsearchRestTemplate.java:1442) ~[spring-data-elasticsearch-3.2.13.RELEASE.jar:3.2.13.RELEASE]
	at org.springframework.data.elasticsearch.core.ElasticsearchRestTemplate.index(ElasticsearchRestTemplate.java:701) ~[spring-data-elasticsearch-3.2.13.RELEASE.jar:3.2.13.RELEASE]
	at org.openl.rules.ruleservice.storelogdata.elasticsearch.ElasticSearchStoreLogDataService.save(ElasticSearchStoreLogDataService.java:151) ~[org.openl.rules.ruleservice.ws.storelogdata.elasticsearch-5.26.0-SNAPSHOT.jar:5.26.0-SNAPSHOT]
	... 20 more
````
It's starting to reproduce after migration to new Jackson Databinding version

Reply:
The test is not passed in jdk11. It is just not run. :)
```xml
            <id>qytqylkage</id>
            <activation>
                <jdk>1.8</jdk>
            </activation>
```